CMakeLists.txt

```diff
@@ -375,7 +374,9 @@ if(BUILD_TEST)
     set_property(SOURCE ${JIT_TEST_SRCS} APPEND PROPERTY COMPILE_OPTIONS "-Werror")
   endif()
 
-  install(TARGETS ${NVFUSER_TESTS} DESTINATION bin)
+  if (NOT PROJECT_IS_TOP_LEVEL)
+    install(TARGETS ${NVFUSER_TESTS} DESTINATION bin)
```

Comment:
nitpick, we can probablyjust put this inside the `PROJECT_IS_TOP_LEVEL` block above.